src/client/refresh/vk/vk_util.c

```diff
@@ -210,7 +210,7 @@ buffer_flush(BufferResource_t *buf)
 	VkResult result = VK_SUCCESS;
 
 	VkMappedMemoryRange ranges[1] = {{
-		.sType = VK_STRUCTURE_TYPE_BUFFER_CREATE_INFO,
+		.sType = VK_STRUCTURE_TYPE_MAPPED_MEMORY_RANGE,
```

Comment:
This doesn't look like debug code. This commit should be merged with one of the previous ones so there are no broken commits in case someone has to bisect.

Reply:
thank you, I have fix to original commit  3dc4fa0